app/vmctl/flags.go

```diff
@@ -524,6 +531,11 @@ var (
 			Usage:    fmt.Sprintf("Split export data into chunks. Requires setting --%s. Valid values are %q,%q,%q,%q.", remoteReadFilterTimeStart, stepper.StepMonth, stepper.StepDay, stepper.StepHour, stepper.StepMinute),
```

Comment:
Wdyt about adding a comment for this flag which will mention that by default migration order is from oldest to newest intervals and reference to new flag to change this behaviour?

Reply:
comment was added